### PR TITLE
Add log readers

### PR DESCRIPTION
**Pull request by J535D165**
Based on the work op @qubixes in simulations project. 

Log readers output `Logger` objects. 

@qubixes Shall we rename `Logger` into `ALLog`. Logger is also used in the default lib `logging` I think.

### CODE REVIEW

**Comment by J535D165** [comment]
@qubixes Can you review this one?

File: asr/logging.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        File path of the log file.
+
+    Returns
+    -------
+    Logger:
+        A Logger object with logs.
+    """
+    log_fp_path = Path(log_fp)
+
+    if log_fp_path.is_file() and log_fp_path.suffix == ".json":
```
Comment (qubixes) [inline asr/logging.py]: I don't think checking for suffix should be necessary? Just try/except when loading the file?

File: asr/logging.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            log = Logger()
+            log._log_dict = json.load(f)
+
+            return log
+
+    else:
+        raise ValueError(f"Incorrect file path {log_fp}")
+
+
+def read_logs_from_dir(log_dir, prefix=None):
```
Comment (qubixes) [inline asr/logging.py]: How about prefix="", then by default it will use all files in the directory?

File: asr/logging.py (hunk 1 above)
Comment (J535D165) [reply asr/logging.py]: Great! Will change this one.

File: asr/logging.py (hunk 2 above)
Comment (J535D165) [reply asr/logging.py]: This is a bug. `""` will fail as well atm. Will fix this one.

**Pull request merged**